### PR TITLE
Revised the division operator article

### PR DESCRIPTION
**Pull request by pkulikov**
Added links to the C# spec and moved the snippets to the samples repo.
Supported by dotnet/samples#505

### CODE REVIEW

File: docs/csharp/language-reference/operators/division-operator.md [hunk 1]
```diff
@@ -9,22 +9,36 @@ helpviewer_keywords:
...
+
+[!code-csharp-interactive[integer division](~/samples/snippets/csharp/language-reference/operators/DivisionExamples.cs#Integer)]
+
+To obtain the quotient of the two operands as a floating-point number, use the `float`, `double`, or `decimal` type:
+
+[!code-csharp-interactive[integer as floating-point division](~/samples/snippets/csharp/language-reference/operators/DivisionExamples.cs#IntegerAsFloatingPoint)]
+
+## Floating-point division
+
+For the `float`, `double`, and `decimal` types, the result of the `/` operator is the quotient of the two operands:
```
Comment (BillWagner) [inline docs/csharp/language-reference/operators/division-operator.md]: I think it is worth mentioning that if one of the operands is `decimal`, the other can't be `float` or `double`. (There is an implicit conversion from integer to decimal, but not from float or double to decimal. You must explicitly convert a float or double argument to decimal.)

**Review by BillWagner — DISMISSED** [review]
This is also great work @pkulikov 

I have one comment that represents a number of questions we've had on the conversions among floating point types.

**Review by BillWagner — CHANGES_REQUESTED** [review]
There's one clarification that would help readers understand the rules on converting flloating point numbers.

File: docs/csharp/language-reference/operators/division-operator.md (hunk 1 above)
Comment (pkulikov) [reply docs/csharp/language-reference/operators/division-operator.md]: @BillWagner thanks for review. I've addressed your feedback.

**Comment by BillWagner** [comment]
closing and reopening to start a new build.

**Review by BillWagner — APPROVED** [review]

**Comment by BillWagner** [comment]
Great updates again @pkulikov 
Thank you.

**Pull request merged**